Features/ProfilePage.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Profile Management
+
+Background:
+    Given I am on the home page
```

Comment:
you should split the features into two feature files one for language and one for skills!

Reply:
Thank you for your feedback! This has been fixed.